core/templates/pages/donate-page/donate-page.module.ts

```diff
@@ -78,17 +90,17 @@ const downgradedModule = downgradeModule(bootstrapFn);
 
 declare var angular: ng.IAngularStatic;
 
-angular.module('oppia', [
-  'dndLists', 'headroom', 'infinite-scroll', 'ngAnimate',
-  'ngAudio', require('angular-cookies'), 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
-  'ngSanitize', 'ngTouch', 'pascalprecht.translate',
-  'toastr', 'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
-  downgradedModule
-])
+angular.module('oppia').requires.push(downgradedModule);
+angular.module('oppia')
   // This directive is the downgraded version of the Angular component to
   // bootstrap the Angular 8.
   .directive(
     'serviceBootstrap',
     downgradeComponent({
       component: ServiceBootstrapComponent
-    }) as angular.IDirectiveFactory);
+    }) as angular.IDirectiveFactory).directive(
+    'donatePage',
+      downgradeComponent({
+        component: DonatePageComponent
+      }) as angular.IDirectiveFactory
+  );
```

Comment:
Should we do this in the `donate-page.component.ts` itself. Because as we migrate more components there would be a long list of these here.

Reply:
Right, done!